Sources/UITableView.swift

```diff
@@ -0,0 +1,83 @@
+import UIKit
+import ReactiveSwift
+import Result
+
+// FIXME: Temporary workaround for the lack of `BindingTarget` without `Lifetime`.
+private let (lifetime, token) = Lifetime.make()
+
+extension NSObject: ReactiveExtensionsProvider {}
+
+extension Reactive where Base: UITableView {
+	public var collection: BindingTarget<(UITableView) -> UITableViewDataSource> {
+		return BindingTarget(lifetime: lifetime) { [weak base] in
```

Comment:
Ideally this would be the lifetime of the table view. But it would require pulling in ReactiveCocoa, or unless we split the project like how RxSwift and RxCocoa do.